content/pt-br/docs/concepts/policy/pid-limiting.md

```diff
@@ -0,0 +1,123 @@
+com alguns riscos adicionais. De qualquer forma, um único Pod não será capaz de trazer
+toda a máquina para baixo. Este tipo de limitação de recursos ajuda a evitar simples
+fork bombs de afetar uma operação de um cluster inteiro.
+
+A limitação de PID por pod permite que os administradores protejam um pod de outro, mas
+não garante que todos os pods alocados nesse host não possam afetar o nó em geral.
+A limitação por pod também não protege os próprios agentes do nó do esgotamento do PID.
+
+Você também pode reservar uma quantidade de PIDs para consumo adicional do nó, separada da
+alocação para os pods. Isso é semelhante a como você pode reservar uma CPU, memória ou outros
+recursos para uso pelo sistema operacional e outras instalações fora dos Pods
+e seus contêineres.
+
+A limitação de PID   é um recurso importante relacionado a [recurso
+computacional](/pt-br/docs/concepts/configuration/manage-resources-containers/), requerimentos
+e limites. No entanto, você o especifica de uma maneira diferente: em vez de definir um
+limite de recursos do pod no `.spec` para um pod, você configura o limite como uma
+configuração no kubelet. Os limites de PID definidos pelo pod não são suportados no momento.
+
+{{< caution >}}
+Isso significa que o limite aplicável a um pod pode ser diferente dependendo
+onde o pod está alocado. Para tornar as coisas simples, é mais fácil se todos os nós usarem
+os mesmos limites e reservas de recursos PID.
+{{< /caution >}}
+
+## Limites de PID de nós
+
+O Kubernetes permite que você reserve vários IDs de processo para uso do sistema. Para
+configurar a reserva, use o parâmetro `pid=<número>` no
+opções de linha de comando `--system-reserved` e `--kube-reserved` para o kubelet.
+O valor especificado declara que o número especificado de IDs de processo será
+reservado para o sistema como um todo e para os daemons do sistema Kubernetes
+respectivamente.
+
+{{< note >}}
+Antes da versão 1.20 do Kubernetes, limitação de recursos PID com nível de nós
+reservas necessárias habilitando o [`feature-gates`
+](/docs/reference/command-line-tools-reference/feature-gates/)
+`SupportNodePidsLimit` para funcionar.
+{{< /note >}}
+
+## Limites de PID do pod
+
+O Kubernetes permite limitar o número de processos em execução em um pod. Você
+especifica esse limite no nível do nó, em vez de configurá-lo como um recurso
+limite para um pod específico. Cada nó pode ter um limite de PID diferente.
+Para configurar o limite, você pode especificar o parâmetro de linha de comando `--pod-max-pids`
+para o kubelet ou definir `PodPidsLimit` no arquivo de configuração do kubelet
+[arquivo de configuração](/docs/tasks/administer-cluster/kubelet-config-file/).
+
+{{< note >}}
+Antes da versão 1.20 do Kubernetes, era necessário habilitar a limitação de recursos PID para pods
+o [portão de recursos](/docs/reference/command-line-tools-reference/feature-gates/)
+`SupportPodPidsLimit` para funcionar.
+{{< /note >}}
+
+## Remoção baseada em PID
```

Comment:
Can we use "Encerramento baseado em PID" here? Maybe use Despejo, but I´d like to hear other people. @stormqueen1990 @MrErlison

Reply:
I personally prefer (and sometimes suggest) using "remoção" because despejo is a bit loaded, especially in the context of Linux (I always think of "despejo de núcleo"/"despejo de memória"). "Encerramento" could be a good fit too but I kind of still lean towards "remoção" because "encerramento" could be a normal termination of a process (whereas eviction is an abnormal termination).